### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by Ivanfifan**
[DEMO LINK](https://Ivanfifan.github.io/layout_catalog/)
[TEST REPORT LINK](https://Ivanfifan.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,351 @@
...
+      <nav class="nav">
+        <ul class="products">
+          <a href="#" class="products__link is-active">Apple</a>
+          <a href="#" class="products__link">Samsung</a>
+          <a href="#" class="products__link">Smartphones</a>
+          <a
+          href="#"
+          class="products__link"
+          data-qa="nav-hover">
+            Laptops &amp Computers
```
Comment (anastasiiavorobiova) [inline src/index.html:33]: Fix indentation accordingly to the code style guide

File: src/styles/card.scss [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    background-color: $blue-color;
+    border: 1px solid $blue-color;
+    border-radius: 5px;
+    text-align: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #fff;
```
Comment (anastasiiavorobiova) [inline src/styles/card.scss:80]: Use variables for colors

File: src/styles/header.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  flex-direction: row;
+}
+
+.products {
+  display: flex;
+
+  &__link {
+    line-height: 60px;
+    text-decoration: none;
+    font-family: Roboto, sans-serif;
```
Comment (anastasiiavorobiova) [inline src/styles/header.scss:20]: Consider adding font-family for body and get all elements inherited from it

File: src/index.html [hunk 4]
```diff
@@ -1,12 +1,351 @@
...
+          <a href="#" class="products__link">Video</a>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div class="card" data-qa="card">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27
+        Retina 5K Monoblock&quot (MNED2UA/A)"
```
Comment (anastasiiavorobiova) [inline src/index.html:47]: Fix indentation accordingly to the code style guide

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,351 @@
...
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+        ref="#buy"
+        class="card__buy"
+        data-qa="card-hover">
```
Comment (anastasiiavorobiova) [inline src/index.html:77]: Fix indentation accordingly to the code style guide

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,351 @@
...
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+        ref="#buy"
+        class="card__buy"
+        data-qa="hover">
```
Comment (anastasiiavorobiova) [inline src/index.html:115]: Fix indentation accordingly to the code style guide

File: src/styles/card.scss [hunk 7]
```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
```
Comment (anastasiiavorobiova) [inline src/styles/card.scss:6]: Consider adding color to variables

File: src/styles/card.scss [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 1.5;
+    margin-bottom: 4px;
+    transition: 300ms;
+  }
+
+  &:hover &__name {
+    color: #34568b;
```
Comment (anastasiiavorobiova) [inline src/styles/card.scss:25]: Consider adding colors to variables

File: src/styles/header.scss [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    text-transform: uppercase;
+    color: #060b35;
+    transition: 300ms;
+
+    &:last-child {
+      margin-right: 0;
+    }
+
+    &:hover {
+      color: #00acdc;
```
Comment (anastasiiavorobiova) [inline src/styles/header.scss:34]: See my comments above

File: src/styles/header.scss [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
```
Comment (anastasiiavorobiova) [inline src/styles/header.scss:41]: See my comments above

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Added some comments, but unfortunately, couldn't open your DEMO link. [Use instructions to publish your site](https://mate-academy.github.io/layout_task-guideline/#important)

File: src/index.html [hunk 11]
```diff
@@ -1,12 +1,349 @@
...
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+        ref="#buy" class="card__buy"
+        data-qa="hover"
```
Comment (anastasiiavorobiova) [inline src/index.html:342]: Add indentations accordingly to the code style guide
```
<a
  ref="#buy" class="card__buy"
  data-qa="hover"
>
```

File: src/index.html [hunk 12]
```diff
@@ -1,12 +1,349 @@
...
```
Comment (anastasiiavorobiova) [inline src/index.html:304]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 13]
```diff
@@ -1,12 +1,349 @@
...
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="/images/logo.png"
+        alt="logo"
+        class="logo__img"
```
Comment (anastasiiavorobiova) [inline src/index.html:21]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 14]
```diff
@@ -1,12 +1,349 @@
...
+        >
+      </a>
+      <nav class="nav">
+        <ul class="products">
+          <a href="#" class="products__link is-active">Apple</a>
+          <a href="#" class="products__link">Samsung</a>
+          <a href="#" class="products__link">Smartphones</a>
+          <a
+          href="#" class="products__link"
+          data-qa="nav-hover"
```
Comment (anastasiiavorobiova) [inline src/index.html:31]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 15]
```diff
@@ -1,12 +1,349 @@
...
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+        ref="#buy" class="card__buy"
+        data-qa="card-hover"
```
Comment (anastasiiavorobiova) [inline src/index.html:74]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 16]
```diff
@@ -1,12 +1,349 @@
...
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+        ref="#buy" class="card__buy"
+        data-qa="hover"
```
Comment (anastasiiavorobiova) [inline src/index.html:112]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 17]
```diff
@@ -1,12 +1,349 @@
...
```
Comment (anastasiiavorobiova) [inline src/index.html:189]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 18]
```diff
@@ -1,12 +1,349 @@
...
```
Comment (anastasiiavorobiova) [inline src/index.html:228]: Add indentations accordingly to the code style guide

File: src/index.html [hunk 19]
```diff
@@ -1,12 +1,349 @@
...
```
Comment (anastasiiavorobiova) [inline src/index.html:266]: Add indentations accordingly to the code style guide

File: src/styles/card.scss [hunk 20]
```diff
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $second-white;
+  border-radius: 5px;
+  transition: 300ms;
```
Comment (anastasiiavorobiova) [inline src/styles/card.scss:8]: Add an empty line here to make your code more readable

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Added tiny comments to fix

File: src/index.html [hunk 21]
```diff
@@ -1,12 +1,348 @@
...
+      </a>
+      <nav class="nav">
+        <ul class="products">
+          <a href="#" class="products__link is-active">Apple</a>
+          <a href="#" class="products__link">Samsung</a>
+          <a href="#" class="products__link">Smartphones</a>
+          <a
+            href="#" class="products__link"
+            data-qa="nav-hover"
+          >
```
Comment (IvanFesenko) [inline src/index.html:31]: Fix all similar cases
```suggestion
          <a
            href="#"
            class="products__link"
            data-qa="nav-hover"
          >
```

File: src/index.html [hunk 22]
```diff
@@ -1,12 +1,348 @@
...
+          <a href="#" class="products__link">Video</a>
+        </ul>
+      </nav>
+    </header>
+    <main class="content">
+      <div class="card" data-qa="card">
+        <img
+          src="../src/images/imac.jpeg" alt="APPLE A1419 iMac 27 Retina 5K Monoblock&quot (MNED2UA/A)"
+          class="card__photo"
+        >
```
Comment (IvanFesenko) [inline src/index.html:46]: Fix all similar cases
```suggestion
        <img
          src="../src/images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock&quot (MNED2UA/A)"
          class="card__photo"
        >
```

File: src/index.html [hunk 23]
```diff
@@ -1,12 +1,348 @@
...
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <a
+          ref="#buy" class="card__buy"
+          data-qa="hover"
+        >
```
Comment (IvanFesenko) [inline src/index.html:74]: Fix all similar cases
```suggestion
        <a
          ref="#buy"
          class="card__buy"
          data-qa="hover"
        >
```

File: src/index.html [hunk 24]
```diff
@@ -1,12 +1,348 @@
...
+        Buy
+        </a>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="../src/images/imac.jpeg"
+          alt="APPLE A1419 iMac 27
+        Retina 5K Monoblock&quot (MNED2UA/A)"
+          class="card__photo"
+        >
```
Comment (IvanFesenko) [inline src/index.html:84]: Fix all similar cases
```suggestion
        <img
          src="../src/images/imac.jpeg"
          alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
          class="card__photo"
        >
```

File: src/styles/header.scss [hunk 25]
```diff
@@ -0,0 +1,53 @@
+.logo__img {
+  padding-left: 50px;
+  margin: 10px 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 60px;
```
Comment (IvanFesenko) [inline src/styles/header.scss:9]: Do not hard code height - use padding for link instead

File: src/styles/stars.scss [hunk 26]
```diff
@@ -0,0 +1,24 @@
...
+  }
+
+  &--5 :nth-of-type(n),
+  &--4 :nth-of-type(-n + 4),
+  &--3 :nth-of-type(-n + 3),
+  &--2 :nth-of-type(-n + 2),
+  &--1 :nth-of-type(-n + 1) {
+    background-image: url(../images/star-active.svg);
+  }
+}
```
Comment (IvanFesenko) [inline src/styles/stars.scss:24]: 2023-06-27_13h24_52

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Please pay attetntion on code style

Also TEST REPORT LINK isn't working

**Review by mykhalenych — CHANGES_REQUESTED** [review]
check your deploy and test links they aren't working

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome!

**Pull request closed without merging**